src/cpp/shared/combat/combat_spells.cpp

```diff
@@ -632,4 +665,459 @@ void combatManager::CastSpell(int proto_spell, int hexIdx, int isCreatureAbility
   }
   WaitEndSample(res, res.sample);
   this->CheckChangeSelector();
+}
+
+CURSOR_DIRECTION combatManager::GetCursorDirection(int screenX, int screenY, int hex) {
+  int offsetX;
+  int offsetY;
+
+  offsetX = screenX - 44 * (hex % 13 - 1) - 67;
+  if ( !(hex / 13 & 1) )
+    offsetX = screenX - 44 * (hex % 13 - 1) - 89;
+  
+  offsetY = screenY - 63 - 42 * (hex / 13) - 26;
+
+  int clockHour = 0;
+  int v15 = offsetX - 22;
+  if(v15 >= 0) {
+    if(offsetY >= 0)
+      clockHour = 6;
+  } else if(offsetY >= 0) {
+    clockHour = 12;
```

Comment:
This is totally the wrong variable name, right?

Reply:
which? clockHour is correct

Reply:
Oh. I see. Maybe rename it to "directionInClockHours". "clockHours" sounded to me like  a unit of time, which is  weird to have here.

Reply:
Somehow I thought it's hours in clock (wtf clock with 24 hours???), in reality it's just 24 parts of hex
Not sure why it's divided in this many in the algo, instead of just 6
Anyway fixed